scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -265,9 +314,9 @@ public void cancel() {
     @Override
     public boolean isCanceled() {
       try (final var connection = dataSource.getConnection()) {
-        return getRequest(connection, specId, specRevision).canceled();
-      } catch (final NoSuchRequestException ex) {
-        throw new Error("Scheduling request no longer exists");
+        return getRequest(connection, specId, specRevision)
+            .map(RequestRecord::canceled)
+            .orElseThrow(() -> new Error("Request corrupted"));
```

Comment:
I like the use of `Optional` here instead of exceptions - could you help me understand the reasoning behind the changed error message?

Before: "Scheduling request no longer exists"
After: "Request corrupted"

Reply:
Weird, I've grepped around trying to find where the `"Request corrupted"` string came from and can't find it. I changed these strings back to their original form.